scripts/start/calculate_CES_configuration.R

```diff
@@ -5,7 +5,7 @@
 # |  REMIND License Exception, version 1.0 (see LICENSE file).
 # |  Contact: remind@pik-potsdam.de
 calculate_CES_configuration <- function(cfg, check = FALSE) {
-    CESstring <- paste0("indu_", cfg$gms$industry,"-",
+    CESstring <- paste0("indu_", cfg$gms$industry,ifelse(cfg$gms$cm_subsec_model_steel=="processes","pbs",""),"-",
```

Comment:
we will try to get rid of this if-clause as soon as possible, means when thsi will be our default :-)

Reply:
Independent of which is default, we have to distinguish between the two as long as 1) this is part of the subsectors realization and 2) we retain both options (processes and ces)?

Reply:
sure, I was in teh far future when there are all parts of industry transfered and our default :-)